androidplot-core/src/main/java/com/androidplot/xy/StepModelFit.java

```diff
@@ -0,0 +1,67 @@
+package com.androidplot.xy;
+
+import com.androidplot.Region;
+
+/**
+ * Created by psi on 23.02.2017.
+ *
+ * Subclass of StepModel that chooses from predefined step values
+ *
+ * Depending on the currently displayed range (by value) choose increment so that
+ * the number of lines is closest to StepModel.value
+ */
+
```

Comment:
extra space should be removed. (not sure the javadoc generator will pick up the comment otherwise)